src/store/modules/DownloadModule.ts

```diff
@@ -302,13 +317,13 @@ function getIndexOfDownloadProgress(allDownloads: DownloadProgress[], downloadId
     return index;
 }
 
-function getOnlyActiveDownloads(downloads: DownloadProgress[]): DownloadProgress[] {
+function filterKeepOnlyActiveDownloads(downloads: DownloadProgress[]): DownloadProgress[] {
```

Comment:
I would make this a function that is passed to `.filter()`, i.e:

```typescript
const isActive = (dl: DownloadProgress) => [DownloadStatusEnum.DOWNLOADING, DownloadStatusEnum.INSTALLING].includes(dl.status);
```

Then the callsites would be something like:

```typescript
state.allDownloads = state.allDownloads.filter(isActive);
```

If you find it still ambiguous whether the filter keeps or removes the active ones, the function could be renamed from `isActive` to `removeInactiveDownloads` oslt, (but then the function by itself is a bit oddly named).